src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -51,7 +51,11 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
   return (
     <div className={containerStyles}>
       <div className="chat-sidebar__header">
-        <div className="chat-sidebar__controller" onClick={toggleSidebar}>
+        <button
+          className="chat-sidebar__controller"
+          onClick={toggleSidebar}
+          type={"button"}
```

Comment:
```suggestion
          type="button"
```

Reply:
@yurii-lubynets fixed in 5f6ea55285f3f28e51ae012db3290ac4694d5c03